beacon-chain/node/registration/p2p.go

```diff
@@ -47,7 +47,7 @@ func readbootNodes(fileName string) ([]string, error) {
 		return nil, err
 	}
 	listNodes := make([]string, 0)
-	err = yaml.Unmarshal(fileContent, &listNodes)
+	err = yaml.UnmarshalStrict(fileContent, &listNodes)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
Same thing here, log an error if this is a `yaml.TypeError` instead of returning an error. 
```suggestion
	if err != nil {
		if _, ok := err.(yaml.TypeError); !ok {
			return nil, err
		} else {
			log.WithError(err).Error("There were some issues parsing the bootnodes from a yaml file.")
		}
	}
```